crates/vault-registry/src/types.rs

```diff
@@ -113,20 +163,22 @@ pub struct SystemVault<AccountId, PolkaBTC> {
     pub to_be_redeemed_tokens: PolkaBTC,
 }
 
-impl<AccountId, BlockNumber, PolkaBTC: HasCompact + Default>
-    Vault<AccountId, BlockNumber, PolkaBTC>
+impl<AccountId, BlockNumber, PolkaBTC: HasCompact + Default, Dot: HasCompact + Default>
```

Comment:
Very minor: capitalize `DOT` for consistency, will make find/replace easier when we generalize.